### ISSUE TITLE
Remove migration message from 0.13

### ISSUE THREAD

**Issue by alexeagle** [bazelbuild/rules_nodejs#910]
Anytime we have a failed module resolution in a nodejs_binary we print
```
WARNING: Due to a breaking change in rules_nodejs 0.13.0, target @build_bazel_rules_nodejs//internal/npm_install:browserify-wrapped_bin
must now declare either an explicit node_modules attribute, or
list explicit deps[] or data[] fine grained dependencies on npm labels
if it has any node_modules dependencies.
See https://github.com/bazelbuild/rules_nodejs/wiki#migrating-to-rules_nodejs-013
```

At this point this guidance is probably more likely to be a false positive and cost the user time diagnosing their problem. 0.13 was released 10 months ago (14 Sep 2018)
